command_cursor.c

```diff
@@ -323,10 +323,6 @@ PHP_METHOD(MongoCommandCursor, valid)
 
 	MONGO_CHECK_INITIALIZED(cmd_cursor->zmongoclient, MongoCommandCursor);
 
-	if (!cmd_cursor->started_iterating) {
-		RETURN_FALSE;
-	}
-
```

Comment:
As discussed in chat, this is OK in the C function, which we'll call anyway.

See https://jira.mongodb.org/browse/PHP-1363 (related to MongoCursor).